packages/web/src/pages/upload-page/forms/AttributionModalForm.tsx

```diff
@@ -234,9 +287,15 @@ const AttributionModalFields = () => {
       >
         <AiAttributionDropdown
           {...aiUserIdField}
-          onSelect={(value: AttributionFormValues[typeof AI_USER_ID]) =>
-            setAiUserId(value)
-          }
+          {...aiUserHelperFields}
+          // @ts-ignore - This is passed down to the dropdown input component that takes error
```

Comment:
we could add `type AiAttributionDropdownProps = ComponentPropsWithoutRef<'input'> & ...`